setup.py

```diff
@@ -883,7 +888,7 @@ def print_box(msg):
             'Topic :: Software Development :: Libraries :: Python Modules',
             'Programming Language :: C++',
             'Programming Language :: Python :: 3',
-        ] + ['Programming Language :: Python :: 3.{}' for i in range(python_min_version[1], 9)],
+        ] + ['Programming Language :: Python :: 3.{}'.format(i) for i in range(python_min_version[1], version_range_max)],
```

Comment:
I'm guessing we can't make this change here since this PR is just cherry-picks?
```suggestion
        ] + [f'Programming Language :: Python :: 3.{i}' for i in range(python_min_version[1], version_range_max)],
```

Reply:
Right, let's just keep this cherry pick as simple as possible.

As well for the `setup.py` we should probably make it as compatible as possible and not use features like f strings there (unless already do)

Reply:
Makes sense, yeah I noticed that `setup.py` doesn't already use `f`-strings anyway so probably best to just leave it as-is in any case.